codex/contracts/market.nim

```diff
@@ -92,7 +92,7 @@ method requestStorage(market: OnChainMarket, request: StorageRequest){.async.} =
   convertEthersError:
     debug "Requesting storage"
     await market.approveFunds(request.price())
-    await market.contract.requestStorage(request)
+    discard await market.contract.requestStorage(request).confirm(1)
```

Comment:
It is probably ok to do `confirm(0)` everywhere; we only want to need sure that it ended up in a block before moving on, so that we can be sure that any queries to the contract state take this transaction into account.

Reply:
Good point.

Reply:
I have hard time understanding what `confirm(0)` should do and if it should even work 😳 Can you please comment on the expected behavior?

Reply:
Never mind, found the implementation and understood it.

Hmm, okey I will make it zero for now and lets revisit it later with the reorg research.